src/test/test_data/annotated_metagenome_assembly_save_39794_22.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
It would be nice to remove 90% of the contigs in the `annotated_metagenome_assembly_check_against.json` file so that the test data is smaller, if that's not too hard. Having more data there won't make the tests more accurate

Reply:
sure, I think i've removed like ~120000 already lol

Reply:
donezo